### ISSUE TITLE
Invalid filtering for multiple token options

### ISSUE THREAD

**Issue by RustedBones** [typelevel/sbt-tpolecat#74]
Let's consider those scalac options:
```scala
  val parallelismOption = new ScalacOption("-Ybackend-parallelism" :: "8" :: Nil)
  val releaseOption = new ScalacOption("-release" :: "8" :: Nil)
```
when both added to `tpolecatScalacOptions ++= Set(parallelismOption, releaseOption)` the resulting `scalacOptions` is invalid:
```sh
[info] * -Ybackend-parallelism
[info] * 8
[info] * -release
[info] * -other-option
```

This is due to the `distinct` in  https://github.com/typelevel/sbt-tpolecat/blob/05fe604cf12482a66dfbb96f550b6cf58d3cb4ff/src/main/scala/io/github/davidgregory084/TpolecatPlugin.scala#L132

Which is applied on individual `ScalacOption` token instead of the full option.
The filtering logic in the plugin does not handle well the case of multiple token options.

**Comment by DavidGregory084** [comment]
Yes you are right @RustedBones - the filtering should have been applied on `ScalacOption`

**Comment by DavidGregory084** [comment]
@RustedBones a fix for this went in on #76. I will try to cut a new release (probably 0.4.0) next week

**Comment by RustedBones** [comment]
That was fast :rocket:  Thank you so much !
